feature-styles/HY_P_WatercoursePersistence.se

```diff
@@ -180,7 +180,7 @@ version="1.1.0">
     </se:Rule>
     <se:Rule>
     <se:Description>
-      <se:Title>waterbodies persistence: ephemeral or dry</se:Title>
+      <se:Title>Waterbodies persistence: ephemeral or dry</se:Title>
```

Comment:
For other cases with different geometries, the geometry was mentioned in the title - maybe align?